Samples/WinMLSamplesGallery/WinMLSamplesGallery/WinMLSamplesGallery.csproj

```diff
@@ -36,18 +40,22 @@
     <Compile Remove="Samples\ImageClassifier - Copy.xaml.cs" />
     <Compile Remove="Samples\ObjectDetector\ImageEffects.xaml.cs" />
   </ItemGroup>
+  <ItemGroup>
+	<Compile Remove="Samples\Batching - Copy.xaml.cs" />

```

Comment:
>Copy.xaml. [](http://example.com/codeflow?start=37&length=10)

whats up with all of these copy tags?

Reply:
Not sure... most of them were already there. I think I had added the Batching one because I though it was necessary but removing now.